src/components/questionary/QuestionaryComponentRegistry.tsx

```diff
@@ -26,17 +26,27 @@ import { sampleDeclarationDefinition } from './questionaryComponents/SampleDecla
 import { shipmentBasisDefinition } from './questionaryComponents/ShipmentBasis/ShipmentBasisDefinition';
 import { textInputDefinition } from './questionaryComponents/TextInput/TextInputDefinition';
 
-export interface FormProps<ValueObjectType> {
-  field: ValueObjectType;
+export type FormChildren<ValueObjectType> = (
+  formikProps: FormikProps<ValueObjectType>
+) => React.ReactNode;
+
+export interface QuestionFormProps {
+  question: Question;
+  closeMe: () => unknown;
```

Comment:
Is the return type here on all functions `unknown` for sure or it is `void`?

Reply:
Unknown because it's a callback function and does not care about the return type